weed/util/config.go

```diff
@@ -40,5 +42,19 @@ func LoadConfiguration(configFileName string, required bool) (loaded bool) {
 	}
 
 	return true
+}
 
+func Config() Configuration {
```

Comment:
may not need this function since it is called just twice.

Reply:
the interface Configuration should be the facade of any configuration (including viper),  it should isolate the viper from out project ( seaweeddfs), we should  encapsulate it completly.
this is my opinion.